tools/run_tests/run_tests.py

```diff
@@ -155,6 +155,8 @@ def configure(self, config, args):
                                                                        self.args.compiler)
 
   def test_specs(self):
+    if self.args.compiler == 'freepass' and self.args.travis:
```

Comment:
Rather than the string literal `'freepass'` appearing several times throughout this module, make it a module-scope constant (`_FREE_PASS`). If possible, make its value just a plain `object` (`_FREE_PASS = object()`) and use `is` rather than `==` when testing for it.

This will help the code take on a shape that better communicates that the free pass is not just another compiler choice.